### DISCUSSION TITLE
ValueError: operands could not be broadcast together with shapes (200,) (800,) (200,)

### DISCUSSION THREAD

**Discussion by Astro-JiangDong** [mboquien/cigale#74] [Q&A] [answered]
After I run 'pcigale run', it went wrong like this:
[error.txt](https://github.com/mboquien/cigale/files/13623922/error.txt)

This is my pcigale.ini file
[pcigale.txt](https://github.com/mboquien/cigale/files/13623929/pcigale.txt)

I wonder what the reason is .

**Comment by mboquien** [comment, marked as answer]
Hello. The reason is that you have some burst ages that are older than the age of the stellar populations.

**Reply by Astro-JiangDong** [reply]
Oh! Thank you very much!
